src/Microsoft.Extensions.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -10,8 +10,17 @@
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Static helper class that provides methods to bind types against configuration.
```

Comment:
Static helper class that allows binding values from configuration to strongly typed objects.